src/runner/at-driver.js

```diff
@@ -32,6 +32,10 @@ export class ATDriver {
   constructor({ socket, log }) {
     this.socket = socket;
     this.log = log;
+    socket.on('message', rawMessage => {
```

Comment:
Should we remove this handler on close?

Reply:
https://github.com/w3c/aria-at-automation-harness/pull/78/commits/b084c0d517f9d9042f05a30862ec4a77ed2242c3